Source/Scene/ShadowMap.js

```diff
@@ -791,7 +1055,20 @@ define([
         }
     };
 
+    ShadowMap.prototype.updatePass = function(context, pass) {
+        if (this._isPointLight && this._usesCubeMap) {
+            this._framebuffer._attachTexture(context, WebGLConstants.COLOR_ATTACHMENT0, this._passFaces[pass]);
```

Comment:
It would be better to have a framebuffer for each face rather than changing color attachments. Binding a framebuffer should be faster than having to re-validate the framebuffer after a new texture is attached.